docs/src/config.ts

```diff
@@ -61,20 +61,24 @@ export const SIDEBAR: Sidebar = {
       { text: 'Architecture', link: 'en/architecture-diagrams' },
     ],
     Components: [
+      { text: 'Loading indicator', link: 'en/media-loading-indicator' },
```

Comment:
can we sort these alphabetically? Would be easier to find components.

Reply:
done